app/src/main/res/values/strings.xml

```diff
@@ -155,9 +155,9 @@
     <!--attendee fragment-->
     <string name="attendee_details">Attendee Details</string>
     <string name="start_text">I accept the</string>
-    <string name="terms_text">terms of service</string>
+    <string name="terms_text">Terms of Service</string>
```

Comment:
It should start with a small letter. Only first letter should be capital of the sentence.

Reply:
> It should start with a small letter. Only first letter should be capital of the sentence.

Ok I corrected that!